lib/ghomfp.gi

```diff
@@ -958,6 +970,18 @@ local H, pres,map,mapi,opt;
   return map;
 end );
 
+#############################################################################
+##
+#M  SimplifiedFpGroup( <FpGroup> ) . . . . . . . . .  sinplify the FpGroup by
```

Comment:
sinplify -> simplify

Reply:
Thanks. This was making the homomorphisms more sinful and sinister.